parts/configure-swarm-cluster.sh

```diff
@@ -275,7 +275,7 @@ fi
 echo "processes at end of script"
 ps ax
 date
-echo "completed mesos cluster configuration"

```

Comment:
Any changes you see in the existing Swarm test templates (in customData) are due to this change.